lib/CustomMySQL.js

```diff
@@ -159,6 +159,13 @@ var CustomMySQL = function () {
                 this._cache[0] = query.text;
                 this._cache.splice(1, 0, query.values);
             }
+            // Check if knex instance
+            else if (this._cache[0] && this._cache[0].toSQL) {
+                    var _query = this._cache[0].toSQL();
+
+                    this._cache[0] = _query.sql;
+                    this._cache.splice(1, 0, _query.bindings);
+                }
```

Comment:
Formatting?

Reply:
lib folder is the result of babel.

Reply:
Hum, generated files shouldn't be checked in? Should be part of postinstall script perhaps, oh well